### PR TITLE
Add contextual popup links

### LINKED ISSUE #509: Contextual popup links show even when sk global fonts and colors are disabled.
These two links should not appear if the experiments for sk fonts and colors are not activated.

Screenshot 2022-06-19 at 00 57 29

### PR DESCRIPTION
**Pull request by lushkant**
closes #509

### CODE REVIEW

File: inc/elementor/js/ang-typography.js [hunk 1]
```diff
@@ -23,32 +23,44 @@ jQuery( document ).ready( function() {
...
 				{
 					name: 'global_colors',
 					title: elementor.translate( 'Edit Style Kit Colors' ),
 					callback: analog.openGlobalColors,
-				},
+				}
+			);
+		}
+
+		if ( parseInt( ANG_Typo.has_sk_fonts ) ) {
```
Comment (mauryaratan) [inline inc/elementor/js/ang-typography.js]: @lushkant shouldn't this evaluate for `Boolean`?

Comment (lushkant) [reply inc/elementor/js/ang-typography.js]: You see that only returns 1 or 0 in string so this works, quite efficiently. I've seen this pattern in checkbox validations so thought why not.

But if you strictly want it to be validated as bool, we can do that too?

Comment (mauryaratan) [reply inc/elementor/js/ang-typography.js]: Could be! But I don't see a reason to treat expected Boolean value as Int?

Comment (lushkant) [reply inc/elementor/js/ang-typography.js]: alright, I'll do that!

Comment (lushkant) [reply inc/elementor/js/ang-typography.js]: done

**Pull request merged**